docs/source/user_guide/model_understanding.ipynb

```diff
@@ -763,6 +792,36 @@
     "\n",
     "The returned `objective_thresholds` dictionary has the objective measure as the key, and the dictionary value associated contains both the best objective score and the threshold that results in the associated score."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize high dimensional data in lower space\n",
+    "\n",
+    "We can use T-SNE to visualize data with many features on a 2D plot, making it easier to see relationships in your data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Our data is highly dimensional, we can't plot this in a way we understand\n",
+    "print(len(X.columns))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.model_understanding import graph_t_sne\n",
+    "\n",
+    "graph_t_sne(X)"
```

Comment:
There's no output here:
<img width="783" alt="image" src="https://user-images.githubusercontent.com/22552445/162017106-16347fe2-7ffc-4ed9-91ee-b5bd8755b331.png">

Filed an issue [here](https://github.com/alteryx/evalml/issues/3447) but likely need to address that first before we can move in with this.

Reply:
So weird and good catch @bchen1116 ! Maybe we can try to build the docs locally to see if they show up in the generated html

Reply:
@bchen1116 since this graphing issue is independent of these changes, not sure it's worth it to block this PR on them getting fixed